azure-pipelines.vrt-pr.yml

```diff
@@ -79,21 +81,13 @@ jobs:
           BASELINE_PIPELINE_ID: $(BASELINE-PIPELINE-ID-WEBCOMPONENTS)
         condition: eq(variables['vrTestSkip'], 'no')
 
-      - bash: node node_modules/vrscreenshotdiff/lib/index.js pr --clientType "FluentUI-web-components" --buildId $(Build.BuildId) --lkgCIBuild $(LatestBuildId) --pipelineId $(pipelineId) --pipelineName '$(pipelineName)'
+      - bash: vr-app run-diff --screenshotsDirectory ./screenshots --buildType pr --clientType "FLUENTUI" --ciDefinitionId 214 --groupName $(pipelineName) --locationPrefix 'FluentUI-web-components' --locationPostfix 'vrscreenshotwebcomponents' --pipelineId $(pipelineId)  --clientName 'fluentui-web-components-v3' --threshold '0.04' --cumThreshold '1'
```

Comment:
Can you please provide more context about this bash command's parameters and what they do? This comment is valid for #28527 #28526 and #28525

Reply:
There is a download-vr-cli.yml template which gets executed before this step. Basically it downloads the vr-app-cli package and lets us run the vr-app command(for the screenshot diffing/baseline upload) where the parameters mean the following:
screenshotsDirectory: location of the screenshots in the agent
buildType: whether its a release build(CI) where baseline upload task is performed or the pr build where diffing needs to be performed.
location prefix, postfix, clientName are the identifying parameters on the diffing tool for the screenshots location in the blob storage.
threshold and cumthreshold are the parameters that the diffing tool uses to set a threshold. If the diff in pixels exceeds the provided threshold, it gets marked as a diff image.